virtual/registry/sql_registry.go

```diff
@@ -0,0 +1,36 @@
+package registry
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	_ "github.com/lib/pq"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+// NewSQLRegistry creates a new SQL backed registry.
+// Currently this only supports Postgres.
```

Comment:
If only Postgres is supported why are you importing the SQLite thing. Is that just because the PR is still WIP?

Reply:
Used for testing removing as it is really a completely different driver with all of the required options to set and query modifications.